app/src/main/java/de/westnordost/streetcomplete/quests/fire_hydrant_ref/AddFireHydrantRef.kt

```diff
@@ -0,0 +1,36 @@
+package de.westnordost.streetcomplete.quests.fire_hydrant_ref
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.quest.NoCountriesExcept
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement
+import de.westnordost.streetcomplete.osm.Tags
+
+class AddFireHydrantRef : OsmFilterQuestType<FireHydrantRefAnswer>() {
+
+    override val elementFilter = """
+        nodes with
+        emergency=fire_hydrant
+        and !name and !ref and noref != yes and ref:signed != no and !~"ref:.*"
+    """
+
+    override val changesetComment = "Determine fire hydrant refs"
+    override val wikiLink = "Key:ref"
+    override val icon = R.drawable.ic_quest_fire_hydrant_ref
+    override val achievements = listOf(EditTypeAchievement.LIFESAVER)
+    override val isDeleteElementEnabled = true
+    override val enabledInCountries = NoCountriesExcept(
+        "CH", "FR"
```

Comment:
As [`b8586d8`](https://github.com/streetcomplete/StreetComplete/pull/4440/commits/b8586d84d9fc5798c2f81e7271ea8057ba73615c) was "just for testing", should Austria be added back?

```suggestion
        "CH", "FR", "AT"
```